quark/core/parallelquark.py

```diff
@@ -113,7 +115,8 @@ def _apply_analysis_result(self, rule_obj):
     def __init__(self, apk, core_library, num_of_process=1):
         self._result_map = {}
         self._pool = Pool(
-            num_of_process - 1, self._worker_initializer, (apk, core_library)
+            min(num_of_process, cpu_count - 1), self._worker_initializer,
```

Comment:
`cpu_count` is a function. You should call it to get the number of cores.

Reply:
thanks, silly mistake.